specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,227 @@
+
+```xml
+<Grid Background="White" Width="150">
+    <ScrollViewer x:Name="Target" Height="300">
+        <TextBlock
+            Foreground="Black"
+            FontSize="30"
+            Text="{x:Bind LoremIpsum}"
+            TextWrapping="Wrap">
+        </TextBlock>
+    </ScrollViewer>
+
+    <MonochromaticOverlayPresenter
+        ReplacementColor="Green"
+        Background="Black"
+        TargetElement="{Binding ElementName=Target}"
+        Height="30"
+        IsHitTestVisible="False"
+        VerticalAlignment="Center"
+        HorizontalAlignment="Stretch"/>
+</Grid>
+```
+
+![Example showing the MonochromaticOverlayPresenter control overlaid on moving content within a ScrollViewer, changing the foreground color and background color of words that pass through.](images/api-example.gif)
+
+The example below shows how the new `MonochromaticOverlayPresenter` API can be used to partially highlight a TextBlock.
+
+```xml
+ <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <StackPanel x:Name="Panel" Orientation="Vertical" HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Grid Margin="2,10,0,0">
+                <TextBlock x:Name="Target" Text="This string will be partially highlighted"/>
+                <MonochromaticOverlayPresenter
+                    ReplacementColor="Black"
+                    TargetElement="{Binding ElementName=Target}"
+                    Width="50"
+                    VerticalAlignment="Stretch"
+                    HorizontalAlignment="Center"
+                    Background="LightBlue"/>
+            </Grid>
+        </StackPanel>
+    </Grid>
+```
+
+> Todo: add a reflection example (classic use of VisualBrush in WPF)
+
+### MonochromaticOverlayPresenter theme resources
+
+You can modify the default Style and ControlTemplate to give the control a unique appearance. 
+For more info, see the 
+[Light-weight styling section](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling)
+of the 
+[Styling controls](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles)
+article.
+
+`MonochromaticOverlayPresenter` defines the following new resource:
+
+```xml
+<StaticResource x:Key="DatePickerFlyoutPresenterHighlightForegroundColor" ResourceKey="TextOnAccentAAFillColorPrimary" />
+```
+
+
+## Other MonochromaticOverlayPresenter members
+
+| Name | Description |
+| - | - |
+| SourceElement | Gets or sets the UIElement to render inside the `MonochromaticOverlayPresenter`. |
+| ReplacementColor | Gets or sets the Color to use instead of the non-transparent pixels of the SourceElement.
+
+
+# API Details
+
+```csharp
+namespace Microsoft.UI.Xaml.Controls.Primitives
+{
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass MonochromaticOverlayPresenter : Windows.UI.Xaml.Controls.FrameworkElement
```

Comment:
we currently inherit from Grid for this class, as we use the Background and CornerRadius properties that don't exist in FrameworkElement or Panel (Background exists but CornerRadius doesn't)

Reply:
@MikeHillberg Are there other types that would be better to derive from? I think Control wouldn't work because Control only has the properties, it doesn't do the drawing.

Reply:
The implementation can derive from Grid, we just don't need/want the public API to have extra unnecessary members.